libs/blocks/video/video.css

```diff
@@ -6,3 +6,98 @@ video {
   max-width: 100%;
   height: auto;
 }
+
+
+.pause-play-wrapper img {
+  min-height:40px !important;
```

Comment:
The `min-height` is set to 150px in marquee.css Line 659  which requires me to set min-height 40px.
!important is required because this overrides my min-height.

Reply:
I'd still advise that we find a way to make the selector more specific or have a specific override in the marquee file, rather than use `!important`